extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.configuration.js

```diff
@@ -505,7 +505,7 @@ angular.module('PaperUI.controllers.configuration',
 		} else {
 		    thing.item = {};
         }
-		thingSetupService.update(thing, function() {
+		thingService.updateConfig({thingUID:thing.UID},thing.configuration, function() {
```

Comment:
Okay, there are save actions for JS but it seems to be not working... Don't know why.